editor/selectors.js

```diff
@@ -3,6 +3,7 @@
  */
 import moment from 'moment';
 import { first, last, get } from 'lodash';
+import { createSelector } from 'reselect';
```

Comment:
Thought the `memoize` call needs to be updated to compute a hash from all the arguments (instead of the first one only)